lib/index.js

```diff
@@ -137,6 +137,7 @@ router.get('/apps/:appId/reviews', function (req, res, next) {
     return apps;
   }
 
+  req.query.sort = gplay.sort[req.query.sort || 'NEWEST']
```

Comment:
don't mutate the query object. This achieves the same effect without doing it:
```
const opts = Object.assign({appId: req.params.appId, sort: 'NEWEST'}, req.query);
```

Reply:
there dose have a problem if mutate the query object.
the next page link will like :
/api/apps/com.elex.lastbattlegroundsurvival/reviews?lang=en&<span style="color:red">***sort=0***</span>&page=1